.gitmodules

```diff
@@ -1,4 +1,4 @@
 [submodule "ext_libs/vowpal_wabbit"]
 	path = ext_libs/vowpal_wabbit
-	url = https://github.com/VowpalWabbit/vowpal_wabbit.git
-	branch = master
+	url = https://github.com/jackgerrits/vowpal_wabbit.git
```

Comment:
Not sure if this is desired change. Ping here to confirm.

Reply:
You're right, this will need to be changed before the PR is merged. I am waiting on the CMake PR to get merged in VW before I can switch this back to using master